CONTRIBUTORS.md

```diff
@@ -0,0 +1,77 @@
+<html>

```

Comment:
I think as of now we would only be looking at *Bug reports, Code, Documentation, Design, Ideas & Planning, Reviewed Pull Requests, Tests* 
These can be automated. Let's start with a basic version, meanwhile we can come up with improvements.